### PR TITLE
Video call state moved to headers

### PR DESCRIPTION
**Pull request by aforge**
Call state/duration moved to replacement msg headers.
    
'webrtc' header contains state (started, accepted, finished, disconnected).
'webrtc-duration' header is for the server-computed call duration.

### CODE REVIEW

File: server/calls.go [hunk 1]
```diff
@@ -52,12 +52,20 @@ type videoCall struct {
...
 	// Call message seq ID.
 	seq int
+	// Call message content.
+	content interface{}
+	// Time when the call was accepted.
+	acceptedAt time.Time
 }
 
-func (call *videoCall) messageHead() map[string]interface{} {
+func (call *videoCall) messageHead(newState string, duration int64) map[string]interface{} {
```
Comment (or-else) [inline server/calls.go:61]: `int64` looks like an overkill. `int` should be plenty enough.

File: server/calls.go [hunk 2]
```diff
@@ -52,12 +52,20 @@ type videoCall struct {
...
 	head := make(map[string]interface{})
 	head["mime"] = constTinodeVideoCallMimeType
```
Comment (or-else) [inline server/calls.go:63]: Remove? Change to Drafty mime?

**Review by or-else — APPROVED** [review]

File: server/calls.go (hunk 1 above)
Comment (aforge) [reply server/calls.go]: Changed.

File: server/calls.go (hunk 2 above)
Comment (aforge) [reply server/calls.go]: Removed.

**Pull request merged**